docs/docsite/rst/user_guide/intro_getting_started.rst

```diff
@@ -79,7 +79,7 @@ Now ping all your nodes:
    $ ansible all -m ping
 
 Ansible will attempt to remote connect to the machines using your current
-user name, just like SSH would.  To override the remote user name, just use the '-u' parameter.
+user name, just like SSH would.  To override the remote user name, just use the '-u' parameter. (Note that if you have specified a user in the inventory file this will unintuitively take precedence over the '-u' parameter. To take precedence over the inventory, use '-e "ansible_ssh_user=<USER>"')
```

Comment:
```suggestion
user name, just like SSH would.  To override the remote user name, just use the '-u' parameter. (Note that if you have specified a user in the inventory file this will  take precedence over the '-u' parameter. To take precedence over the inventory, use '-e "ansible_ssh_user=<USER>"'. See :ref:`general_precedence_rules` for details.)
```

Reply:
I'd personally leave 'unintuitively' in there. I'm aware from many other issues this is a bone of contention. The precedence in place is intentional and I get that. But it's very unusual to not have command line args override and so it is unintuitive. I wasted a lot of time finding this out.